### ISSUE TITLE
Unhelpful error message during task construction

### ISSUE THREAD

**Issue by mxm0** [ros-planning/moveit_task_constructor#67]
We had the following error when using the task constructor. The error was due to a stage "connector" type which was missing information from the stage below. The error message was not really helpful and redrawing the task structure in the command line would be more helpful.

```
[ERROR] [1541083635.557176600]: Error initializing stage:
task pipeline: output interface of 'move to pre-pour pose' doesn't match mine
```

@v4hn

**Comment by rhaschke** [comment]
> redrawing the task structure in the command line would be more helpful

This can be easily achieved like this:
```
Task t;
std::cerr << t << std::endl;
```

**Comment by v4hn** [comment]
> This can be easily achieved like this

Yes, maybe, I didn't check yet.
For newcomers the error message is rather more confusing than necessary and I proposed filing this issue so that we can improve the reported error later.

**Comment by rhaschke** [comment]
What kind of msg do you propose?

**Comment by v4hn** [comment]
Ideally, we might print the structure of the task and add a highlight on the conflicting stage interfaces.

**Comment by rhaschke** [comment]
> Ideally, we might print the structure of the task and add a highlight on the conflicting stage interfaces.

That's what `std::cerr << t << std::endl;` does. But I think, this should be added at a higher level.

**Comment by rhaschke** [comment]
Addressed by #71.
